### ISSUE TITLE
FontAwesome is affected by migration

### ISSUE THREAD

**Issue by PaulPCIO** [ant-design/codemod-v4#116]
Here is an example pre-migration

```jsx
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faChevronRight} from '@fortawesome/pro-light-svg-icons'

const MyCustomIcon = () => {
  return <FontAwesomeIcon icon={faChevronRight} />
}
```

And here is the result post-migration
```jsx
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faChevronRight} from '@fortawesome/pro-light-svg-icons'
import { Icon as LegacyIcon } from '@ant-design/compatible'

const MyCustomIcon = () => {
  return <FontAwesomeIcon icon={<LegacyIcon type={faChevronRight} />} />
}
```

as you can see the FontAwesomeIcon component also gets effected, I believe because of the **icon** prop?

**Comment by vagusX** [comment]
I check the code, and run codemod for your code too. But I cannot repeat the changes you described. Would you like to provide a simplest sample project?

**Comment by PaulPCIO** [comment]
I just recreated an example, and it seems to be fixed after the @1.1.0 update. 👍
